app/serializers/current_user_serializer.rb

```diff
@@ -336,4 +337,8 @@ def experimental_sidebar_enabled
   def include_experimental_sidebar_enabled?
     SiteSetting.enable_experimental_sidebar
   end
+
+  def status
```

Comment:
Same concern as https://github.com/discourse/discourse/pull/16875/files#r878953293https://github.com/discourse/discourse/pull/16875/files#r878953293

Reply:
This also fixed.